master/pkg/model/experiment_config.go

```diff
@@ -80,10 +80,12 @@ func (d *DeviceConfig) UnmarshalJSON(data []byte) error {
 	return errors.Wrap(json.Unmarshal(data, DefaultParser(d)), "failed to parse device")
 }
 
-// ResourcesConfig configures resource usage for an experiment, command, notebook, or tensorboard.
+// ResourcesConfig configures resource usage for an experiment, command, notebook, tensorboard
+// or generic task.
 type ResourcesConfig struct {
 	Slots int `json:"slots"`
 
+	SlotsPerTask   *int         `json:"slots_per_task"`
```

Comment:
I've always found `slots`, `slots_per_task`, and `slots_per_trial` confusing. Is there any way we can just use slots for this?